backend/src/services/collectionService.ts

```diff
@@ -511,7 +511,7 @@ export class CollectionService extends LoggerBase {
 
       const { logoUrl, github, website, twitter } = orgDetail
```

Comment:
We should probably make this conditional (1 repo vs multiple repos).
If 1 repo get logoUrl, github, website and twitter from the repo specifically.
If multiple repos get logoUrl, github, website and twitter from organization.